### PR TITLE
Add docs for starferry upgrade

### PR DESCRIPTION
**Pull request by elise-ng**
Ref oursky/likecoin-chain#310

Confirmed ram required with @hochiw, 32GB ram + 32GB swap would be fine; ref oursky/likecoin-chain#286

### CODE REVIEW

File: validator/starferry-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+- Upgrade cosmos-sdk to 0.45.6, ibc-go to 2.3.0
+- Introduce x/likenft module, and integrate x/nft module backported from cosmos-sdk 0.46.0-rc1
+
+### Timeline
+
+| Event                                                            | Time         |
+| ---------------------------------------------------------------- | ------------ |
+| Testnet Proposal                                                 | W4 June 2022 |
+| Testnet Deploy                                                   | W1 July 2022 |
+| [Mainnet Proposal](https://likecoin.bigdipper.live/proposals/46) | W1 July 2022 |
```
Comment (rickmak) [inline validator/starferry-upgrade.md]: The link is laichikok.

File: validator/starferry-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+#### Recommended
+
+- 8 CPU Cores
+- 32GB RAM + 32GB Swap
+- 500GB+ SSD
+
+#### Minimum
+
+- 4 CPU Cores
+- 16GB RAM + 48GB Swap
```
Comment (rickmak) [inline validator/starferry-upgrade.md]: I think @hochiw comfirm < 32GB instance kill.

**Review by rickmak — COMMENTED** [review]
I will update the doc directly with my changes.

File: validator/starferry-upgrade.md (hunk 2 above)
Comment (hochiw) [reply validator/starferry-upgrade.md]: it's without swap tho, RAM + Swap > 32gb should be okay

File: validator/starferry-upgrade.md (hunk 1 above)
Comment (elise-ng) [reply validator/starferry-upgrade.md]: removed

File: validator/starferry-upgrade.md (hunk 2 above)
Comment (elise-ng) [reply validator/starferry-upgrade.md]: arguably it will be super slow at 16 + 48 even if it doesn't crash... so maybe set minimum to 32+32?

Comment (elise-ng) [reply validator/starferry-upgrade.md]: my concern is community reaction tho

**Comment by rickmak** [comment]
Updated at https://github.com/likecoin/docs/commit/656deb295e6e62de718b9ca3d059a2b72dbf4555

**Pull request closed without merging**